### DISCUSSION TITLE
How to have product in "New World" like lenge House

### DISCUSSION THREAD

**Discussion by TheNak976** [naverz/zepeto-studio-kor#347] [5. Q&A]
I need to know this features from lenge house.

If someone has a precise and detailed answer I'm interested.
Don't tell me that this is in the official guide because I didn't find it except how to create some basic purchasing functions with "IWP".

But what I want to know is how to synchronize with the products of my store and the function that makes that once clicked we can wear them directly to see the look that it gives

https://www.youtube.com/watch?v=Lqou-O12OgY

Thx in advance

**Comment by jsedwardyou** [comment]
I believe there's a bit of confusion between a World created by Zepeto and a World created by creators. World created by creators (also known as a world created from Unity) does not support products that are registered in "World" created by Zepeto team as Worlds from Unity only supports items that are created by creators themselves. 
Please feel free to ask any additional questions you might have regarding IWP or New World System :)

**Reply by TheNak976** [reply]
Okay, now I'll speak the same language as you **"New World"**.

Indeed what I am looking for is a feature that is in the "New World" of **Lenge House**. 

How I can add **my own creations (clothes, bags etc.)** in my **"New World"** so that people can buy and wear them. As you can see in the "New World" of **Lenge House** => https://go.zepeto.me/BFc5O2
![image](https://user-images.githubusercontent.com/16490986/153350208-9b2c10b5-713f-4ccc-8938-f4984a94ba75.png)

Lenge has items **(clothes, accessories etc.)** in her **"New World"** that you can buy and wear directly.
![image](https://user-images.githubusercontent.com/16490986/153350328-44403a41-fd5f-473f-bf7b-f917ec02ae79.png)

So I am **looking for this feature to be able to sell my creations in my "New World"**. Not just an IWP feature where we can make a buy button that is not synced with our own creations.

Or if we can sync IWP with our own creations (clothes, bags etc.), I would like to know how to do it in detail.

I think I have detailed enough, I hope you understood what I wanted.

**Comment by TheNak976** [comment]
Because we agree that by following your guide here https://studio.zepeto.me/guides/creating-a-world, we can create our **"New World"** from unity using the template you provide. But how to add and sell our creations, that's the question?

**Reply by jsedwardyou** [reply]
I recommend you to take a look at these documentations in order:
1. Registering Products (your creations) to Zepeto Studio
[https://studio.zepeto.me/guides/making-world-products](https://studio.zepeto.me/guides/making-world-products)

2. Syncing the products to your IWP button
[https://studio.zepeto.me/guides/in-world-purchases](https://studio.zepeto.me/guides/in-world-purchases)

By following these documents, you should be able to sell your creations to other Zepeto users.
For the shop UI, we currently don't provide any standard formats so it is up to your imaginations and skills to come up with a custom shop that supports your creations.

We look forward to seeing your cool shop soon :)

**Reply by TheNak976** [reply]
Finally you are no help, because this functionality of linking to the zepeto store is available for some users like Lenge and his "Lenge House". But not available in your documentation for public users.

Basically we need to sign a partnership with you to provide us with this item linking functionality.

All we can do with the world products available in your doc is just game pass or bonus items.
But we can't sell our own creations of items indeed. ^^

I even went so far as to search in your library(*.dll) in unity and found out how to add items etc. But how to find the ids of the items available in the zepeto store? No idea ^^.

The world is crazy ^^
